iroh/src/client.rs

```diff
@@ -871,6 +871,11 @@ impl Entry {
         self.0.key()
     }
 
+    /// Get the timestamp of this entry.
+    pub fn timestamp(&self) -> u64 {
```

Comment:
I'm rather sad this comes as a u64.  Perhaps that's already a mistake in iroh-sync?  Or just implementation details leaking out?  But for a user-level Rust API I'd much rather see something like `time::OffsetDateTime` or so.